vsphere-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -9,15 +9,15 @@
 #vsphere_port: 443
 #vsphere_datacenter_name: Datacenter
 #vsphere_resource_pool_name: Resources
-#vsphere_auto_placement: true
+#vsphere_auto_placement: false
```

Comment:
just wondering.. are these changes from "true" to "false" and the likes related to the fact that they were wrong before?